string_manipulation/helper.py

```diff
@@ -0,0 +1,53 @@
+
+
+def remove_multy_occurence(multy_occuring_string):
+    string_after_single_occurence = ""
+
+    for index_for_multy_occurence in multy_occuring_string:
+        if index_for_multy_occurence.lower() not in string_after_single_occurence.lower():
+            string_after_single_occurence = string_after_single_occurence + index_for_multy_occurence
+
+    return string_after_single_occurence
+
+
+def repeats(repeating_character_string, key_character):
```

Comment:
What does `repeats` function do? function name should reflect whats going on inside.